libguard/guard_file.cpp

```diff
@@ -88,15 +83,15 @@ void GuardFile::write(const uint64_t pos, const void* src, const uint64_t len)
                   "Unable to move to the position in the guard file."
                   " Position= 0x%016llx",
                   pos);
-        throw std::runtime_error("Failed to move to the position in the guard file.");
+        throw InvalidEntityPath(
```

Comment:
Why throwing `InvalidEntityPath` exception when failed to seek the guard file?

Reply:
Added GuardSeekFailed.